stubs/core.stub

```diff
@@ -70,10 +70,16 @@ function bin2hex(string $string): string {}
 
 /**
  * @param array<mixed> $result
- * @param-out array<int|string, string|list<string>> $result
+ * @param-out array<int|string, string|list<string>|array<mixed>> $result
  */
 function parse_str(string $string, array &$result): void {}
```

Comment:
moved `parse_str` and `mb_parse_str` next to each other, and equalized the type signatures